### PR TITLE
1.7.2

### PR DESCRIPTION
**Pull request by SpicyFerret**
Bomb rework, works better and properly.

### CODE REVIEW

File: replacers.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"VALUES":{"318128519890862091":{"a":"b ","teste":"testado "}}}
```
Comment (bignelli) [inline replacers.json]: Put this file in .gitignore.

File: src/main/java/nukeologist/kregbot/data/map/RPGMap.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package nukeologist.kregbot.data.map;
+
+public class RPGMap {
+
+
+
+}
```
Comment (bignelli) [inline src/main/java/nukeologist/kregbot/data/map/RPGMap.java]: ?

File: src/main/java/nukeologist/kregbot/data/map/RPGStructures.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package nukeologist.kregbot.data.map;
+
+public class RPGStructures {
+
+}
```
Comment (bignelli) [inline src/main/java/nukeologist/kregbot/data/map/RPGStructures.java]: ?

File: src/main/java/nukeologist/kregbot/data/map/RPGTerrain.java [hunk 4]
```diff
@@ -0,0 +1,4 @@
+package nukeologist.kregbot.data.map;
+
+public class RPGTerrain {
+}
```
Comment (bignelli) [inline src/main/java/nukeologist/kregbot/data/map/RPGTerrain.java]: ?

**Comment by SpicyFerret** [comment]
random stuff deleted...

**Comment by bignelli** [comment]
What is tempImage?

**Comment by SpicyFerret** [comment]
> What is tempImage?

random image generate by the Image class. Can be deleted, but the image class will overwrite it anyway...

**Pull request merged**